src/dataconnect/fileUtils.ts

```diff
@@ -106,3 +106,23 @@ export async function pickService(
   }
   return serviceInfo;
 }
+
+// case insensitive indicators for supported app platforms
+const WEB_INDICATORS = ["package.json", "package-lock.json"];
+const IOS_INDICATORS = ["info.plist", "project.pbxproj", "podfile", "package.swift"];
+const ANDROID_INDICATORS = ["androidmanifest.xml", "build.gradle"];
+
+// given a directory, determine the platform type
+export async function getPlatformFromFolder(dirPath: string) {
+  const fileNames = await fs.readdir(dirPath);
```

Comment:
We probably want to dive down into child directories here.

Reply:
I'm still not certain how much we want to dive down. We are not supporting multiple app detections yet, right?

Reply:
Yes, but the user might have their app multiple directories in

Reply:
I think we went back and forth on this a bit in the meeting, but it is probably fine to omit downward recursion for now.